modulefiles/wcoss2-run.lua

```diff
@@ -0,0 +1,16 @@
+help([[
+]])
+
+local intel_ver=os.getenv("intel_ver") or "19.1.3.304"
+
+load(pathJoin("intel", intel_ver))
+
+
+prepend_path("MODULEPATH", "/apps/test/lmodules/core/")
+load ("GrADS/2.2.2")
```

Comment:
Should grads_ver be used here to follow other templates?

Reply:
Not yet.  GrADS isn't yet in the stack on wcoss2.  When it does arrive I figure to move the grads load to common-run.lua.